Description: Disable oom_instead_of_bump_pointer_overflow test
 Upstream has agreed with  my determination that it is the test rather than the
 code it tests which  is broken, but we have not yet reached consensus on how
 best to fix it. See https://github.com/fitzgen/bumpalo/issues/128
Author: Peter Michael Green <plugwash@debian.org>

--- rust-bumpalo-3.7.0.orig/tests/tests.rs
+++ rust-bumpalo-3.7.0/tests/tests.rs
@@ -58,7 +58,7 @@ fn can_iterate_over_allocated_things() {
     assert!(seen.iter().all(|s| *s));
 }
 
-#[test]
+/*#[test]
 #[should_panic(expected = "out of memory")]
 fn oom_instead_of_bump_pointer_overflow() {
     let bump = Bump::new();
@@ -79,7 +79,7 @@ fn oom_instead_of_bump_pointer_overflow(
 
     // This should panic.
     bump.alloc_layout(layout);
-}
+}*/
 
 #[test]
 fn force_new_chunk_fits_well() {
